### ISSUE TITLE
[Bug] Update Badge component screen

### ISSUE THREAD

**Issue by ludovic35** [Orange-OpenSource/ouds-ios#840]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

all

### OS Version

all

### App version

0.17.0

### App build number

1752049652

### OUDS iOS library version embeded in the app

0.17.0

### App build type

ALPHA

### Describe the issue

Commentaire de @B3nz01d sur la pull request idoine que je recopie ici :

@ludovic35 Here are my feedbacks:
1- to be coherent with the illustration, the default state of the badge component when you arrive on it the first time should be in:

 - type: Count
- size: medium
- number: "1" <- it needs to be defaulted to 1, it cannot stay empty
- Status: negative
- 
2- When a user is set to xsmall or small and selects the icon or the count type the size should switch to medium automatically

3- when you are on the medium setup and you select the small or xsmall it should switch to the corresponding size in the standard type.

4- the selection of the status should use the same drop down as it is used in the divider

**Comment by pylapp** [comment]
@ludovic35 @B3nz01d  Attention, on doit **avant tout** respecter les specifications Figma, notamment sur les valeurs par défaut.
Dans la mesure où l'app de démo doit refléter la lib, et que la lib doit appliquer les règles Figma, il faut donc que l'état par défaut du badge soit _neutral_ (de mémoire, mais presque certain que ce n'est pas _negative_ dans Figma), et que l'app de démo s'y tienne.

**Comment by ludovic35** [comment]
> [@ludovic35](https://github.com/ludovic35) [@B3nz01d](https://github.com/B3nz01d) Attention, on doit **avant tout** respecter les specifications Figma, notamment sur les valeurs par défaut. Dans la mesure où l'app de démo doit refléter la lib, et que la lib doit appliquer les règles Figma, il faut donc que l'état par défaut du badge soit _neutral_ (de mémoire, mais presque certain que ce n'est pas _negative_ dans Figma), et que l'app de démo s'y tienne.

@pylapp t'as dégainé plus vite que moi. J'ai tout expliqué dans le commentaire si dessous

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the new release from GitHub and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **0.17.0**
- Build number: **1755795708**
- Build tag: ****


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
